.ops/ecamp3-logging/files/kibana/kibana-objects.ndjson

```diff
@@ -0,0 +1,146 @@
+{
+  "attributes": {
+    "fieldAttrs": "{\"container.name\":{\"count\":2},\"kubernetes.labels.app_kubernetes_io/name\":{\"count\":2},\"kubernetes.labels.app_kubernetes_io/part-of\":{\"count\":1},\"message\":{\"count\":3},\"json.httpRequest.requestUrl\":{\"count\":1},\"kubernetes.deployment.name\":{\"count\":2},\"kubernetes.labels.app.kubernetes.io/instance\":{\"count\":1},\"kubernetes.pod_name\":{\"count\":1},\"log\":{\"count\":1}}",
+    "fieldFormatMap": "{}",
+    "fields": "[]",
+    "name": "log-index-pattern",
+    "runtimeFieldMap": "{}",
+    "sourceFilters": "[]",
+    "timeFieldName": "@timestamp",
+    "title": "logstash-*",
+    "typeMeta": "{}"
+  },
+  "coreMigrationVersion": "8.8.0",
+  "created_at": "2024-04-23T18:24:23.081Z",
+  "id": "e270616c-823f-485b-b1e5-0d3435383b91",
+  "managed": false,
+  "references": [],
+  "sort": [
+    1713898129206,
+    18
+  ],
+  "type": "index-pattern",
+  "typeMigrationVersion": "8.0.0",
+  "updated_at": "2024-04-23T18:48:49.206Z",
+  "version": "WzYyMSw0XQ=="
+}
+{
+  "attributes": {
+    "columns": [
+      "kubernetes.labels.app.kubernetes.io/instance",
+      "kubernetes.pod_name",
+      "json.httpRequest.requestUrl",
+      "json.httpRequest.escapedUrl",
```

Comment:
I think this should be `"escapedUrl"` instead of `"json.httpRequest.escapedUrl"` (same for the next line)

Reply:
This may be.
I just dumped the state, restored it, and looked if everything worked.
I will dump and restore again.